### ISSUE TITLE
Testing on custom questions/queries

### ISSUE THREAD

**Issue by Param-Raval** [ryanzhumich/editsql#20]
I was able to train/test on the given Spider and "spider_data_removefrom" sets but I want to test the model on a given question-query. I tried editing the queries in the dev.json file but the predictions remain the same as the queries in the original dev.json. Can you help me out?

**Comment by gouldju1** [comment]
+1

**Comment by Akshaysharma29** [comment]
Hi, @Param-Raval have you solved this issue as I am facing the same issue(on Sparc dataset).

**Comment by Param-Raval** [comment]
Yes, I was able to get a workaround. There are certain folders that need to be deleted in order for the model to register the changes. Delete these and run the training script again:

- logs_sparc_editsql
- processed_data_sparc_removefrom
- processed_data_sparc_removefrom_test

After the preprocessing steps, these will appear again but with the changes you made in the json file.

**Comment by Akshaysharma29** [comment]
Hi @Param-Raval thanks for the response but the issue is due to OOV as described in the below issue 
https://github.com/ryanzhumich/editsql/issues/21
Hence Solving OOV issue has solved this issue.

**Comment by imanuelroz** [comment]
@Akshaysharma29 Hi I tryied the adaptation proposed by Param but as I add new data I am facing the problem that you mentioned with OOV, did you manage to solve it?
